cloudformation/master-server-substack.cfn.yaml

```diff
@@ -341,6 +367,36 @@ Resources:
         NetworkInterfaces:
           - NetworkInterfaceId: !Ref 'MasterENI'
             DeviceIndex: 0
+          - !If
+            - UseNIC1
+            - DeviceIndex: 1
+              NetworkCardIndex: 1
+              InterfaceType: !Ref 'AWS::NoValue'
+              Groups: !Ref 'MasterSecurityGroups'
+              SubnetId: !Ref 'MasterSubnetId'
+            - !Ref 'AWS::NoValue'
+          - !If
+            - UseNIC2
+            - DeviceIndex: 2
+              NetworkCardIndex: 1
+              InterfaceType: !If
+                - EnableEFA
```

Comment:
I'm afraid there is some inconsistency in the way we manage EFA on master node.
We should probably never enable EFA for now on head node while as it is now it looks like even if you specify compute you get NIC0 always without EFA and 1-3 with EFA enabled.

Reply:
Done